run.py

```diff
@@ -342,9 +342,7 @@ def _check_port_conflicts(self) -> list[str]:
 
         if self.args.get("use_flask_web_client"):
             if self.is_port_open("localhost", self.args["neuro_san_web_client_port"]):
-                port_conflicts.append(
-                    f"Flask web client port {self.args['neuro_san_web_client_port']} is already in use."
-                )
+                port_conflicts.append(f"Flask web client port {self.args['neuro_san_web_client_port']} is already in use.")
```

Comment:
You need to run `make test` to reformat the files. This line is 124 char long, while our rules say each line should be less than 120. So `make test` will reformat it.